### PR TITLE
Portfolio desktop version

### PR DESCRIPTION
**Pull request by naji-97**
### **The following requirements has benn made:**
- [ ]  I used media queries in my CSS file to make my desktop version look like the template you selected (Desktop: 768px or wider).
- [ ] In order to lay out the elements on the page I used Grid and Flexbox. in accordance with Figma Template

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -430,3 +429,161 @@ div.border-bootom {
...
+  .contac-card {
+    grid-column: 2/4;
+  }
+
+  .contact-me-form .form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+  }
```
Comment (mckent05) [inline style.css:586]: - Good job on the work done so far, but there are some differences between your project and the chosen design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(Please pay attention to the marked section in the Figma template attached below):
- The background images in the section is pixelated, please try to fix this 🙏 
- The section contains the whole width of the screen, there are no white borders on the left and right as can be seen in your design, please fix this in your design 👍 👍 

eththet

Figma design template:
hwrwrg

File: style.css [hunk 2]
```diff
@@ -430,3 +429,161 @@ div.border-bootom {
...
+
+  .languages:first-child {
+    padding: 5px 20px;
+  }
+
+  .about {
+    margin: 30px 5px;
+    background-color: #fff;
+    padding: 20px 16px;
+  }
```
Comment (mckent05) [inline style.css]: - Good job on the work done so far, but there are some differences between your project and the chosen design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

- Your project(Please pay attention to the marked section in the Figma template attached below):

- The images for the listed technology are not visible on my side, a possible cause for this is that you are navigating to the `assets` directory wrongly, I suggest using `./assets/...`. Please fix this 😄 
- The `See Resume` button is not styled as it is in the given Figma template, please fix this 🙏 
- The top right of the section has a border-radius which is absent in your design, please fix this 👍 
- Please fix all the angled arrows for the technologies and make sure they are the same as the given design template 👍 👍 
ad23545y

Figma Design:
rtvrvrtv

File: style.css [hunk 3]
```diff
@@ -430,3 +429,161 @@ div.border-bootom {
...
+
+  ul.links {
+    margin-top: 0;
+  }
+
+  .main-section {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    padding: 50px 130px;
+  }
```
Comment (mckent05) [inline style.css]: - Good job on the work done so far, but there are some differences between your project and the chosen design template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project(Please pay attention to the marked section in the Figma template attached below):

- This section is missing background images as seen in the Figma template, please fix this 🙏. To fix this issue, I advise you consider removing the `background-position: center` property used in the mobile version 
- The bottom left part of this section has a border-radius which is absent in your design, please fix this 😄 
tntenwrynw

Figma template:
t44ttt



The See Resume button is not styled as it is in the given Figma template, please fix this 🙏

The top right of the section has a border-radius which is absent in your design, please fix this 👍

Please fix all the angled arrows for the technologies and make sure they are the same as the given design template 👍 👍

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @naji-97, 

# STATUS: CHANGES REQUIRED ♻️ ♻️ 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👍 👍 
- You have used the right workflow for this milestone 👍 👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (naji-97) [reply style.css]: for the icon I used font-awsome icons maybe you need to reload your page

File: style.css (hunk 1 above)
Comment (Abenezer-Tilahun) [reply style.css:586]: Fixed.

File: style.css (hunk 2 above)
Comment (Abenezer-Tilahun) [reply style.css]: Fixed.

File: style.css (hunk 3 above)
Comment (Abenezer-Tilahun) [reply style.css]: Fixed.

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @naji-97 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️Good Job  Implement the desktop version with the usage of media queries.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**